### ISSUE TITLE
HTML

### ISSUE THREAD

**Issue by Pfeffi1999** [Pfeffi1999/AP-2017#1]
<!DOCTYPE html>
<html>
<head>
<title>jquery Slider</title>
<meta charset="utf-8">
<link href="css/main.css" type="text/css" rel="stylesheet" />
<link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,900&amp;subset=latin-ext" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Dancing+Script:400,700&amp;subset=latin-ext" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Amatic+SC:400,700&amp;subset=latin-ext" rel="stylesheet">
<link rel="icon" type="image/png" href="images/favicon.png" sizes="16x16">
</head>
<body>
<div id="header-wrapper">
  <header class="bg-img">
    <nav>
      <ul>
        <li class="logo"><a href="jq Slider.html" title="Grün Kram &rsaquo;"><img src="images/logo_gruenkram.png" alt="Grün Kram" /></a></li>
        <li><a href="#" title="Veranstaltung &rsaquo;">Veranstaltung</a></li>
        <li><a href="#" title="Anmeldung &rsaquo;">Anmeldung</a></li>
        <li><a href="#" title="Rezepte &rsaquo;">Rezepte</a></li>
        <li><a href="W3-Modul/w3modul.html" title="Bio-Kiste &rsaquo;">Bio-Kiste</a></li>
        <li><a href="#" title="Kontakt &rsaquo;">Kontakt</a></li>
      </ul>
    </nav>
    <div class="burger-menu">
      <div class="logo"><a href="jq Slider.html" title="Grün Kram &rsaquo;"><img src="images/logo_gruenkram.png" alt="Grün Kram" /></a></div>
      <div class="dropdown">
        <button onclick="burgerNavi()" class="dropbtn">
        <div class="bar"></div>
        <div class="bar"></div>
        <div class="bar"></div>
        </button>
        <div id="myDropdown" class="dropdown-content"> <a href="#" title="Veranstaltung &rsaquo;">Veranstaltung</a> <a href="#" title="Anmeldung &rsaquo;">Anmeldung</a> <a href="#" title="Rezepte &rsaquo;"> Rezepte </a><a href="W3-Modul/w3modul.html" title="Bio-Kiste &rsaquo;"> Bio-Kiste </a><a href="#" title="Kontakt &rsaquo;">Kontakt </a></div>
      </div>
    </div>
    <div class="text">
      <h2>"Grünkram" präsentiert</h2>
      <h1>Cooking Night at Greenhouse</h1>
    </div>
  </header>
</div>
<div class="wrapper-content">
  <h2 class="margin-top">Worauf Sie sich freuen können</h2>
  <div class="content-wrapper content-background">
    <div class="content-text-wrapper">
      <div class="content-text-wrap">
        <p>Ein gemeinsam zubereitetes schmackhaftes Menü mit frischen Zutaten aus unserem eigenen Anbau – ökologisch, regional und erschwinglich.</p>
      </div>
      <div class="content-text-wrap">
        <p>Sie erleben Spaß und Genuss beim gemeinsamen Kochen und Schlemmen. Lassen Sie sich überraschen, welche Kochkoryphäe in Ihrer Cooking Night die Anleitung übernimmt!</p>
      </div>
      <div class="content-text-wrap">
        <p>Außerdem eröffnen Sie sich und Ihren Mitbewohner/innen eine<br />
          große Chance: Alle Teilnehmer der Cooking Night können unsere wöchentliche Gemüsekiste drei Monate lang zum Sonderpreis abonnieren.</p>
      </div>
      <div class="content-text-wrap">
        <p><span style="font-weight: 700;">Eingeladen zur Cooking Night sind:</span> <br />
          Studierende der Hochschulorte Worms, Mannheim, Ludwigshafen, Heidelberg und Darmstadt aller Fachrichtungen.</p>
      </div>
    </div>
  </div>
</div>
<div class="wrapper-content">
  <h2>Rezepte zum Warmkochen</h2>
</div>
<div class="slider js-slider">
  <div class="slider-stage">
    <div class="slide-wrapper js-slide-wrapper">
      <div class="slide js-slide">
        <div class="slide-content">
          <div class="slide-description">
            <h2>Gebackene Schwarzwurzeln</h2>
            <button class="js-show-content" data-show-content="schwarzwurzeln" data-button-text="Rezept anzeigen &rsaquo;" data-button-active-text="Rezept ausblenden &rsaquo;">Rezept anzeigen &rsaquo;</button>
          </div>
          <img src="images/gebackene_schwarzwurzel.jpg" alt="Gebackene Schwarzwurzeln" /> </div>
      </div>
      <div class="slide js-slide">
        <div class="slide-content">
          <div class="slide-description">
            <h2>Grünkohlaufstrich</h2>
            <button class="js-show-content" data-show-content="gruenkohlaufstrich" data-button-text="Rezept anzeigen &rsaquo;" data-button-active-text="Rezept ausblenden &rsaquo;">Rezept anzeigen &rsaquo;</button>
          </div>
          <img src="images/gruenkohlaufstrich.jpg" alt="Grünkohlaufstrich" /> </div>
      </div>
      <div class="slide js-slide">
        <div class="slide-content">
          <div class="slide-description">
            <h2>Pastinaken-Suppe</h2>
            <button class="js-show-content" data-show-content="pastinakensuppe" data-button-text="Rezept anzeigen &rsaquo;" data-button-active-text="Rezept ausblenden &rsaquo;">Rezept anzeigen &rsaquo;</button>
          </div>
          <img src="images/pastinakensuppe.jpg" alt="Pastinaken-Suppe"> </div>
      </div>
    </div>
  </div>
  <div class="slide-arrow-navigation">
    <div class="arrow arrowLeft js-arrow-left">&lang;</div>
    <div class="arrow arrowRight js-arrow-right">&rang;</div>
  </div>
  <div class="slide-dot-navigation js-dot-navigation"> </div>
</div>
<div class="rezepte js-rezepte">
  <div class="wrapper-content rezept-item js-show-rezept" data-content="schwarzwurzeln">
    <div class="content-wrapper content-background2">
      <h2>Gebackene Schwarzwurzeln</h2>
      <div class="content-text-wrapper">
        <div class="content-inner-wrapper">
          <div class="content-text-wrap">
            <p class="mT">Zutaten</p>
          </div>
          <div class="content-text-wrap">
            <ul>
              <li>500 g Schwarzwurzeln</li>
              <li>1 Ei</li>
              <li>6 EL Mehl</li>
              <li>1 Glas Weißwein</li>
              <li>Petersilie</li>
              <li>Zitrone</li>
              <li>Salz und Pfeffer</li>
              <li>Olivenöl</li>
            </ul>
          </div>
        </div>
        <div class="content-inner-wrapper">
          <div class="content-text-wrap">
            <p class="mT">Zubereitung</p>
          </div>
          <div class="content-text-wrap">
            <p>Wurzeln schälen, in etwa 4 cm lange Stücke schneiden und bissfest weich dünsten. Das Ei verquirlen, mit Mehl, Wein, fein gehackter Petersilie und geriebener Zitronenschale zu einem dünnflüssigen Teig verrühren. Salzen und pfeffern. Wurzeln durch den Teig ziehen und in der Pfanne in heißem Öl ausbacken. Mit Zitronenscheiben servieren. </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class="wrapper-content rezept-item js-show-rezept" data-content="gruenkohlaufstrich">
    <div class="content-wrapper content-background2">
      <h2>Grünkohlaufstrich</h2>
      <div class="content-text-wrapper">
        <div class="content-inner-wrapper">
          <div class="content-text-wrap">
            <p class="mT">Zutaten</p>
          </div>
          <div class="content-text-wrap">
            <ul>
              <li>250 g Grünkohl</li>
              <li>250 g Hüttenkäse</li>
              <li>2 EL Olivenöl</li>
              <li>2 EL geröstete Sesampaste</li>
              <li>200 g Gerstenflocken</li>
              <li>Paprika</li>
              <li>Kräutersalz</li>
            </ul>
          </div>
        </div>
        <div class="content-inner-wrapper">
          <div class="content-text-wrap">
            <p class="mT">Zubereitung</p>
          </div>
          <div class="content-text-wrap">
            <p>Grünkohl fein hacken, in wenig Wasser garen und abtropfen lassen. Mit Hüttenkäse, Öl, Sesampaste und Gerstenflocken zu einer geschmeidigen Masse rühren. Mit Paprika und Kräutersalz kräftig abschmecken und in ein verschließbares Gefäß füllen. Im Kühlschrank aufbewahren.</p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div class="wrapper-content rezept-item js-show-rezept" data-content="pastinakensuppe">
    <div class="content-wrapper content-background2">
      <h2>Pastinaken-Suppe</h2>
      <div class="content-text-wrapper">
        <div class="content-inner-wrapper">
          <div class="content-text-wrap">
            <p class="mT">Zutaten</p>
          </div>
          <div class="content-text-wrap">
            <ul>
              <li>600 g Pastinaken</li>
              <li>300 g Kartoffeln</li>
              <li>2 l Wasser</li>
              <li>200 g Sahne</li>
              <li>75 g Butter</li>
              <li>2 EL Gemüsebrühe</li>
              <li>Salz und Pfeffer</li>
            </ul>
          </div>
        </div>
        <div class="content-inner-wrapper">
          <div class="content-text-wrap">
            <p class="mT">Zubereitung</p>
          </div>
          <div class="content-text-wrap">
            <p>Kartoffeln und Pastinaken schälen und in Würfel schneiden. Etwa 25 Minuten in der Gemüsebrühe kochen, anschließend pürieren. Mit Salz und Pfeffer abschmecken und leicht aufkochen. Anschließend Butter und Sahne dazu geben und kurz ziehen lassen.</p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
<div class="wrapper-content">
  <h2 class="margin-top">Anmeldung</h2>
  <div class="content-wrapper content-background">
    <div class="content-text-wrapper">
      <div class="content-inner-wrapper">
        <div class="content-text-wrap">
          <p>4 × in diesem Herbst<br>
            jeweils ab 18:30 Uhr<br>
            in unserem Gemüsehof in <br>
            Bobenheim-Roxheim, Viehweg 17 <br>
            <span style="font-weight: 700; display: block; margin-top: 1.5rem;">Unkostenbeitrag: EUR 20,– </span><br />
            <span style="font-weight: 700;">Anmeldeschluss: 1. Oktober 2017</span></p>
        </div>
      </div>
      <div class="content-inner-wrapper">
        <div class="content-text-wrap">
          <form>
            <div class="wrapper-form">
              <label for="name">Name</label>
              <input type="text" placeholder="Name">
            </div>
            <div class="wrapper-form">
              <label for="vorname">Vorname</label>
              <input type="text" placeholder="Vorname">
            </div>
            <div class="wrapper-form">
              <label for="anschrift">Anschrift</label>
              <input type="textarea" rows="3" placeholder="Anschrift">
            </div>
            <div class="wrapper-form">
              <label for="email">E-Mail</label>
              <input type="email" placeholder="E-Mail">
            </div>
            <div class="wrapper-form">
              <label for="telefon">Telefon</label>
              <input type="tel" placeholder="Telefon">
            </div>
            <div class="wrapper-form">
              <label for="studienfach">Studienfach</label>
              <input type="text" placeholder="Studienfach">
            </div>
            <div class="wrapper-form">
              <label for="studienort">Studienort</label>
              <select>
                <option selected>Bitte wählen Sie aus ...</option>
                <option>Hochschulorte Worms</option>
                <option>Hochschulorte Mannheim</option>
                <option>Hochschule Ludwigshafen</option>
                <option>Hochschule Heidelberg</option>
                <option>Hochschule Darmstadt</option>
              </select>
            </div>
          </form>
        </div>
      </div>
      <div class="content-inner-wrapper">
        <div class="content-text-wrap">
          <h3>Lieblingstermin <br />
            für die Cooking Night</h3>
        </div>
        <div class="content-text-wrap">
          <form>
            <label for="Freitag Oktober">
              <input type="radio" name="lieblingstermin">
              Freitag, 27. Oktober 2017</label>
            <br />
            <label for="Samstag Oktober">
              <input type="radio" name="lieblingstermin">
              Samstag, 28. Oktober 2017</label>
            <br />
            <label for="Freitag November">
              <input type="radio" name="lieblingstermin">
              Freitag, 3. November 2017</label>
            <br />
            <label for="Samstag November">
              <input type="radio" name="lieblingstermin">
              Samstag, 4. November 2017</label>
          </form>
        </div>
      </div>
      <div class="content-inner-wrapper">
        <div class="content-text-wrap">
          <h3>Ersatztermin<br/>
            für die Cooking Night</h3>
        </div>
        <div class="content-text-wrap">
          <form>
            <label for="Freitag Oktober">
              <input type="radio" name="lieblingstermin">
              Freitag, 27. Oktober 2017</label>
            <br />
            <label for="Samstag Oktober">
              <input type="radio" name="lieblingstermin">
              Samstag, 28. Oktober 2017</label>
            <br />
            <label for="Freitag November">
              <input type="radio" name="lieblingstermin">
              Freitag, 3. November 2017</label>
            <br />
            <label for="Samstag November">
              <input type="radio" name="lieblingstermin">
              Samstag, 4. November 2017</label>
          </form>
        </div>
        <div class="content-text-wrap"></div>
        <div class="content-text-wrap">
          <button class="absenden">Absenden &rsaquo;</button>
        </div>
      </div>
    </div>
  </div>
</div>
<div class="wrapper-content">
  <h2 class="margin-top">Unsere Bio-Kiste für Sie</h2>
  <div class="content-wrapper content-background">
    <div class="content-text-wrapper">
      <div class="content-text-wrap"> <img src="images/biokiste.jpg" alt="Bio-Kiste"> </div>
      <div class="content-text-wrap">
        <form>
          <div class="wrapper-form2">
            <label for="name">Name</label>
            <input type="text" placeholder="Name">
          </div>
          <div class="wrapper-form2">
            <label for="vorname">Vorname</label>
            <input type="text" placeholder="Vorname">
          </div>
          <div class="wrapper-form2">
            <label for="anschrift">Anschrift</label>
            <input type="textarea" rows="3" placeholder="Anschrift">
          </div>
          <div class="wrapper-form2">
            <label for="lieferadresse">Lieferadresse</label>
            <input type="textarea" rows="3" placeholder="Lieferadresse">
          </div>
          <div class="wrapper-form2">
            <label for="rechnungsadresse">Rechnungsadresse</label>
            <input type="textarea" rows="3" placeholder="Rechnungsadresse">
          </div>
          <div class="wrapper-form2">
            <label for="email">E-Mail</label>
            <input type="email" placeholder="E-Mail">
          </div>
          <div class="wrapper-form2">
            <label for="telefon">Telefon</label>
            <input type="tel" placeholder="Telefon">
          </div>
          <div class="label-width">
          <label for="kistengroesse">Kistengröße:</label>
          <br />
          <input type="radio">
          <label for="Einpersonenhaushalt"> Einpersonenhaushalt</label>
          <br />
          <input type="radio">
          <label for="Zweipersonenhaushalt"> Zweipersonenhaushalt</label>
          <br />
          <input type="radio">
          <label for="Mehrpersonenhaushalt"> Mehrpersonenhaushalt</label>
          <br />
          <label for="Spezial-Abo">
            <input type="checkbox">
            Ich will das Spezial-Abo für die Gemüsekiste in Anspruch nehmen. </label>
          <br />
          <label for="Spezial-Abo">
            <input type="checkbox">
            Ich will mich erst auf der Cooking Night entscheiden, 
            ob ich das Sonder-Abo in Anspruch nehme. </label>
          <br />
			</div>
          <button class="absenden">Absenden &rsaquo;</button>
        </form>
      </div>
    </div>
  </div>
</div>
<footer>
  <div class="wrapper-content">
    <div class="content-wrapper content-background2">
      <div class="content-text-wrapper">
        <div class="content-inner-wrapper">
          <div class="content-text-wrap">
            <h2>Kontakt und Info-Telefon</h2>
          </div>
          <div class="content-text-wrap">
            <p>U. Grünting<br />
              cookingnight@gruenkram.eu <br />
              062397/9876-12</p>
          </div>
          <div class="content-text-wrap" style="margin-top: 2rem;"></div>
          <div class="content-text-wrap"> <a href="https://de-de.facebook.com/" target="_blank"><img src="images/facebook.png" alt="Facebook"></a> <a href="https://www.snapchat.com/l/de-de/" target="_blank"><img src="images/snapchat.png" alt="Snapchat"></a> <a href="https://www.pinterest.de/" target="_blank"><img src="images/pinterest.png" alt="Pinterest"></a> </div>
        </div>
        <div class="content-inner-wrapper">
          <div class="content-text-wrap">
            <h2>Impressum</h2>
          </div>
          <div class="content-text-wrap">
            <p>Gemüsehof „Grünkram“ <br />
              Uwe Grünting, <br />
              Thea Schwarzheide <br />
              Viehweg 17 <br />
              67240 Bobenheim-Roxheim</p>
          </div>
          <div class="content-text-wrap">
            <p><span style="font-weight: 700; display: block; margin-top: 1.5rem;">Telefon:</span> 062397/9876-0 <br />
              <span style="font-weight: 700; display: block; margin-top: 1.5rem;">E-Mail: </span>u.gruenting@gruenkram.eu <br />
              www.gruenkram.eu</p>
          </div>
          <div class="content-text-wrap"></div>
          <div class="content-text-wrap">
            <p><span style="font-weight: 700; display: block; margin-top: 1.5rem;">Registergericht:</span> Amtsgericht Worms <br />
              Registernummer:<br />
              HRB 12345 <br />
              Umsatzsteuer-Identifikationsnummer gemäß §27a Umsatzsteuergesetz: <br />
              DE 123456789 <br />
              Steuer-Nummer: 098 765 432 1</p>
          </div>
        </div>
      </div>
    </div>
  </div>
</footer>
<script src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.2.1.min.js"></script> 
<script src="js/index.js" type="text/javascript"></script>
</body>
</html>
